### ISSUE TITLE
Linklerin snapshot'larının alınması

### ISSUE THREAD

**Issue by shibby** [linkfloyd/linkfloyd-php#21]
Bu işlem için @miratcan tarafından bir cli projesi geliştirilecek. Burada dursun bu issue.

**Comment by miratcan** [comment]
@shibby bu iş #9 numaralı işin yapılması için mi gerekli?

**Comment by shibby** [comment]
hayır. sayfanın görselini alabiliyoruz zaten. bu ileride yapılacak bi iş. linkleri aynı zamanda snapshot olarak kaydederiz. senin düşündüğün projeyi burada test etmiş oluruz.

**Comment by miratcan** [comment]
tamamdır

**Comment by miratcan** [comment]
Bu işe de gerek yok abi, ben bunu sabitle adında başka bir repoda zaten yapıyorum onun işlerini orada ayrı olarak tutuyorum. Bunun burada durmasına gerek yok ilerde implemente edilmesi için iş açarız. İtirazın yoksa bu işi de kapatıyorum?

**Comment by shibby** [comment]
İş kalabilir bence. milestone'da olmasına gerek yok. İleride yapılacaklar arasında dursun. Eğer diğer repo public ise buradan referanslayabilirsin. Durmasının zararı yok :)
